lib/pry/pry_instance.rb

```diff
@@ -529,28 +529,29 @@ def should_print?
   # @return [String] The prompt.
   def select_prompt
     object = current_binding.eval('self')
-
     open_token = @indent.open_delimiters.any? ? @indent.open_delimiters.last :
       @indent.stack.last
 
-    c = Pry::Config.from_hash({
```

Comment:
Should we delete the definition of `Pry::Config.from_hash`?

Reply:
I don't know what you mean. Delete `from_hash` ? No. It's intentionally recursive.